### ISSUE TITLE
Version 7 unknown

### ISSUE THREAD

**Issue by ryan-merrell** [Qowyn/ark-tools#20]
I recently tried to export tamed dino data and got the error regarding Version 7 being unsupported, so I came and got your latest stable release (0.5.3) and that fixed the issue.

Now I'm trying to do the feed command and I'm getting: 
```
./ark-tools.sh feed /home/steam/ragnarok/ShooterGame/Saved/SavedArks/Ragnarok.ark /home/steam/ragnarok/ShooterGame/Saved/SavedArks/Ragnarok.tempfed
java.lang.UnsupportedOperationException: Version 7 is unknown and cannot be written in binary form
        at qowyn.ark.ArkSavegame.calculateHeaderSize(ArkSavegame.java:477)
        at qowyn.ark.ArkSavegame.writeBinary(ArkSavegame.java:319)
        at qowyn.ark.tools.EditingCommands.feed(EditingCommands.java:101)
        at qowyn.ark.tools.App.main(App.java:117)
```

**Comment by ryan-merrell** [comment]
If it helps, I have java version 1.8.0_131 from ubuntu. Ark map is Ragnarok running on v264.15

**Comment by stealthgyro** [comment]
I wanted to mention I had this same issue trying to import a dino after exporting and editing the JSON.  For myself I'm on Java 8.0.1440.1 on Windows 10.

**Comment by TyrelSackett** [comment]
I posted earlier today on the forums. Same issue trying to convert Json to Map.

**Comment by stealthgyro** [comment]
@TyrelSackett I saw your post on their as well, I figured the issue tracker would be a little less noisy than a 9 page thread. I will not his commit for 0.5.3 only says "Early Support for loading V7 savegames" so I'm sure he's aware, I would love to contribute, but my java is rusty and I don't even know where to begin with some of these files.

**Comment by Qowyn** [comment]
Analyzing the unknown stuff V7 to V9 is the next thing on my todo list, expect a new version maybe this weekend

**Comment by Qowyn** [comment]
Should be completely fixed with the 0.6.0 release
